src/components/NewMessage/NewMessage.vue

```diff
@@ -516,7 +516,7 @@ export default {
 
 			if (this.upload) {
 				// Clear input content from store
-				this.$store.dispatch('setCurrentMessageInput', { token: this.token, text: '' })
+				this.chatExtrasStore.setChatInput({ token: this.token, text: '' })
```

Comment:
Maybe it's better to remove it from the store completely and evaluate against its presence ? 
![image](https://github.com/nextcloud/spreed/assets/84044328/2d3720be-f9b7-4528-a3c5-6c496f5407d5)

Reply:
Empty value is the default behaviour for every chat opening. So it doesn't make sense to delete reactive variable from the store and put it back again. In addition, we:ll never get a null as a value for NcRichContenteditable

Reply:
I checked, it's okay.